### DISCUSSION TITLE
About How Cache Works

### DISCUSSION THREAD

**Discussion by ghost** [mienaiyami/yomikiru#19] [Q&A] [answered]
Previously, to reduce the memory usage you disabled caching of the previous chapter after going to the next chapter and it worked.

Q) How does the application handle the present viewing folder?
- If a folder has 100+ images does the application load them all?

**Comment by mienaiyami** [comment, marked as answer]
Yes, currently it does load all images at once so reader can have lag free experience while scrolling.

**Comment by omodia** [comment]
I tested further by opening the next chapter immediately after the current chapter is loaded. Did this for 4 chapters.
The RAM usage increased, which is normal. But it did not decrease completely.
The decrease in RAM is for 1 chapter. I think it cached the previous 3 chapters.
Can you check it?

**Reply by mienaiyami** [reply]
I did my best to clear as much as cache as possible but its not possible to change or remove cache directly in this framework.
As of now, only way to do this is to refresh the app itself.
Im working on improving it.
